var/spack/repos/builtin/packages/miopen-opencl/package.py

```diff
@@ -167,7 +147,10 @@ class MiopenOpencl(CMakePackage):
         "5.3.0",
         "5.3.3",
         "5.4.0",
+        "5.4.3",
     ]:
+        depends_on("rocm-cmake@%s:" % ver, type="build", when="@" + ver)
+        depends_on("rocm-opencl@" + ver, when="@" + ver)
```

Comment:
Curious.  Are these changes -- removing the versions from above and adding the (same) dependencies here -- intended to make it easier to "deprecate" the older versions?  Or more for documentation purposes?

Reply:
Basically, i just felt i could combine both for loops and still have the dependencies build correctly.